src/pyAMReX.cpp

```diff
@@ -83,6 +83,7 @@ PYBIND11_MODULE(amrex_3d_pybind, m) {
                PODVector
                StructOfArrays
                Vector
+               Utility
```

Comment:
Keeping alphabetic order :)
```suggestion
               Utility
               Vector
```